styles.css

```diff
@@ -0,0 +1,84 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  width: 22.2%;
```

Comment:
- [ ] Please note that you are to stick to design as much as possible. Kindly make your mobile version to look like the figma design. YOu can use 'dev tools' to check what it looks like on your browser. Consider removing this width and I think it will look fine.
### Your design
![t1](https://user-images.githubusercontent.com/94551484/191843372-82d64a57-d7d7-409a-945e-0c08ee725ec6.png)

### Figma template
![tf1](https://user-images.githubusercontent.com/94551484/191843441-e4992992-571c-462b-a483-6da16dd42ef5.png)